### DISCUSSION TITLE
Have you receieved my email there is my pubilc key?

### DISCUSSION THREAD

**Discussion by Matt0244** [AnjaliSheshappa/Tafe22S104#3] [General]
Hi Anjali,

Have you received my email there is my public key.?  That can be used in the future.

**Comment by AnjaliSheshappa** [comment]
Hi, Matt

I have recieved

**Comment by AnjaliSheshappa** [comment]
Hi Matt,

Have you finished your coding

**Comment by Matt0244** [comment]
I am doing now
